miniwin/miniwin/src/miniwin.cpp

```diff
@@ -21,6 +23,7 @@ ULONG IUnknown::Release()
 
 HRESULT IUnknown::QueryInterface(const GUID& riid, void** ppvObject)
 {
+	SDL_LogError(LOG_CATEGORY_MINIWIN, "IUnknown does not implement guid");
```

Comment:
SDL3 has a `SDL_GUIDToString` function.
Perhaps we should introduce a little error function that prints the guid?

Reply:
That would be useful I ended up making a big list to figure out some of the ones that need implemented to get to this point